qchem/pennylane_qchem/qchem/structure.py

```diff
@@ -506,25 +655,25 @@ def generate_hamiltonian(
 
     **Example usage:**
 
-    >>> H, n_qubits = generate_hamiltonian('h2', 'h2.xyz', 0, 1, 'sto-3g')
+    >>> H, n_qubits = generate_hamiltonian('h2', 'h2.xyz', 0, 1, 'sto-3g', wires=['w0','w1','w2','w3'])
     >>> print(n_qubits)
     4
     >>> print(H)
-    (-0.04207897647782188) [I0]
-    + (0.17771287465139934) [Z0]
-    + (0.1777128746513993) [Z1]
-    + (-0.24274280513140484) [Z2]
-    + (-0.24274280513140484) [Z3]
-    + (0.17059738328801055) [Z0 Z1]
-    + (0.04475014401535161) [Y0 X1 X2 Y3]
-    + (-0.04475014401535161) [Y0 Y1 X2 X3]
-    + (-0.04475014401535161) [X0 X1 Y2 Y3]
-    + (0.04475014401535161) [X0 Y1 Y2 X3]
-    + (0.12293305056183801) [Z0 Z2]
-    + (0.1676831945771896) [Z0 Z3]
-    + (0.1676831945771896) [Z1 Z2]
-    + (0.12293305056183801) [Z1 Z3]
-    + (0.176276408043196) [Z2 Z3]
+    (-0.04207897647782188) [Iw0]
+    + (0.17771287465139934) [Zw0]
+    + (0.1777128746513993) [Zw1]
+    + (-0.24274280513140484) [Zw2]
+    + (-0.24274280513140484) [Zw3]
+    + (0.17059738328801055) [Zw0 Zw1]
+    + (0.04475014401535161) [Yw0 Xw1 Xw2 Yw3]
+    + (-0.04475014401535161) [Yw0 Yw1 Xw2 Xw3]
+    + (-0.04475014401535161) [Xw0 Xw1 Yw2 Yw3]
+    + (0.04475014401535161) [Xw0 Yw1 Yw2 Xw3]
+    + (0.12293305056183801) [Zw0 Zw2]
+    + (0.1676831945771896) [Zw0 Zw3]
+    + (0.1676831945771896) [Zw1 Zw2]
+    + (0.12293305056183801) [Zw1 Zw3]
+    + (0.176276408043196) [Zw2 Zw3]
```

Comment:
Cool. BTW, now the Hamiltonian looks really ininteligible :-)

Reply:
Yes. I thought about adding a `_`, but perhaps not necessary for now.

Reply:
Imagine if we label the qubits with our names, we will have terms like `ZZy XMaria YAlain` ... :-))